src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/AssignableUserSearchStep.java

```diff
@@ -60,12 +60,7 @@ public String getFunctionName() {
     @Override
     public String getDisplayName() {
       return getPrefix()
-          + "Searches assignable JIRA Users by username, name or email address for the given project/issueKey";
-    }
-
-    @Override
-    public boolean isMetaStep() {
-      return true;
+              + "Searches assignable JIRA Users by username, name or email address for the given project/issueKey";
```

Comment:
Why reindentation?

Reply:
Argh, IntelliJ did that automatically...

Reply:
And fixed/force-pushed.